script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -1022,7 +1022,7 @@ The first one is that they must be wealthy.
 [sha:83380ef9ab7b6e8944a05c12a6d5112e9d61e885]{
 -- Page 732, Offset 26260.
 -- 魔術のような学んで修得する神秘ではなく、生まれついてその肉体が持ってしまう特異能力。
-I'm not talking about things you can learn, like the Mysteries of Magecraft,
+I'm not talking about mysteries you can learn, like magecraft,
```

Comment:
since this is a proper noun, I'm guessing its a nasuverse term

Reply:
revert, Mysteries of Magecraft is correct. It's a lore term

Reply:
I think the emphasis here shouldn't be on "the Mysteries of Magecraft" because that makes it sound like that's a term unto itself when what is really being said here is that magecraft is a form of mystery (神秘) which can be taught.

If you want to make this easier to think about, another common translation for 神秘 is "arcane". 
"I'm not talking about arcana you learn, such as magecraft..." would be the case then.

("Then why not use that term?" Because mystery has been so established by now its used in the official English language stuff inside the El-Melloi anime)